src/store/plugins/initSdk.js

```diff
@@ -130,6 +130,9 @@ export default (store) => {
         // TODO: remove after resolving https://github.com/aeternity/ae_mdw/issues/508
         spec.paths['/name/pointees/{id}'] = spec.paths['/names/pointees/{id}'];
         delete spec.paths['/names/pointees/{id}'];
+        // TODO: remove after resolving https://github.com/aeternity/ae_mdw/issues/1029
+        spec.paths['/txs/{direction}'] = spec.paths['/txs'];
```

Comment:
as discussed in https://github.com/aeternity/ae_mdw/issues/1029 I think this has already been fixed, we will deploy today